front-end/src/app/reports/transactions/transaction-container/transaction-container.component.ts

```diff
@@ -20,11 +20,24 @@ export class TransactionContainerComponent extends DestroyerComponent {
       if (this.transaction) {
         const title: string = this.transaction.transactionType?.title ?? '';
         this.titleService.setTitle(title);
+      } else {
+        throw new Error('Fecfile: No transaction found in TransactionContainerComponent');
       }
     });
   }
 
-  isDoubleTransaction() {
-    return !!this.transaction?.transactionType?.dependentChildTransactionType;
+  isSingleTransaction(): boolean {
+    return !this.transaction?.transactionType?.dependentChildTransactionTypes?.length;
+  }
+
+  isDoubleTransaction(): boolean {
+    return this.transaction?.transactionType?.dependentChildTransactionTypes?.length === 1;
+  }
+
+  isTripleTransaction(): boolean {
+    if (this.transaction?.transactionType?.dependentChildTransactionTypes) {
```

Comment:
The LOAN_RECEIVED_FROM_BANK, which has 2 dependent child transaction types, C1_LOAN_AGREEMENT and LOAN_RECEIVED_FROM_BANK_RECEIPT, and will resolve to true. The 3-transaction input form has 2 dependent children. I think it would be less ambiguous if changed to "=== 2", though

Reply:
Resolved. Could remove the if statement when the >== was changed to ===